src/main/java/org/mariadb/jdbc/internal/MariaDbServerCapabilities.java

```diff
@@ -69,7 +69,7 @@
   public static final int SSL = 2048; /* Switch to SSL after handshake */
   public static final int IGNORE_SIGPIPE = 4096; /* IGNORE sigpipes */
   public static final int TRANSACTIONS = 8192;
-  public static final int RESERVED = 16384; /* Old flag for 4.1 protocol  */
+  public static final int SERVER_SESSION_STATE_CHANGED = 16384; /* Old flag for 4.1 protocol  */
```

Comment:
This is not the right place to define this flag. It's not part of the "server capabilities" but part of the "server status", and already defined here: https://github.com/mariadb-corporation/mariadb-connector-j/blob/3bc66153b51aca188afc50ff35a0123f16c099ed/src/main/java/org/mariadb/jdbc/internal/util/constant/ServerStatus.java#L69

Reply:
Thanks a lot for your review. I've updated the code.